Model/Person/Tagging/Builder.php

```diff
@@ -74,17 +75,20 @@ class Builder extends PersonBuilder
      * @param NostoLogger $logger
      * @param EventManager $eventManager
      * @param NostoHelperData $nostoHelperData
+     * @param NostoHelperCustomer $nostoHelperCustomer
      */
     public function __construct(
         GroupRepository $groupRepository,
         CustomerRepositoryInterface $customerRepository,
         NostoEmailRepository $emailRepository,
         NostoLogger $logger,
         EventManager $eventManager,
-        NostoHelperData $nostoHelperData
+        NostoHelperData $nostoHelperData,
+        NostoHelperCustomer $nostoHelperCustomer
     ) {
         $this->groupRepository = $groupRepository;
         $this->customerRepository = $customerRepository;
+        $this->nostoHelperCustomer = $nostoHelperCustomer;
```

Comment:
This injection will become redundant after generation is placed into a static util